sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -491,7 +532,7 @@ def convert_feature(fx):
         target_idx=target_idx,
         pdp_lim=pdp_lim,
         deciles=deciles,
-        kind=kind,
+        kind=kind_,
```

Comment:
I think this can be considered BC breaking. In 1.0 when `kind` is a string, `PartialDependenceDisplay.kind` is a string. With this PR `kind` is always a list.

If we want BC, I think we can pass `kind` directly here:

```suggestion
        kind=kind,
```

It looks like `plot` already correctly handles this.

Reply:
I did not think about this one. Right, it makes sense. I will add a small test then.